### ISSUE TITLE
Cloud password reports as incorrect

### ISSUE THREAD

**Issue by nerotrobe** [JurajNyiri/HomeAssistant-Tapo-Control#564]
### Description

When attempting to setup I get the following error every time: 

Invalid cloud password. Make sure you are entering the password for your cloud account and NOT the password which you created via the camera settings (unless they are the same). You need to enter password which you used with your email when signing into the Tapo app.

I've double checked and the password I am providing is 100% the password that is needed to log into the Tapo app.

### Reproduction Steps

1. When requested input the IP address for the camera.
2. When requested input the Camera Account login/password.
3. When requested input the cloud password.
4. Note that the system returns that the cloud password is incorrect.

### Expected behavior

It's expected that the system not report a correct password as incorrect

### Device Firmware

1.3.11

### Integration Version

5.4.17PSA

### Using stream component

No

### Does camera work via official integrations?

Yes

### Camera has all attributes filled out in developer tools

No

### HASS Environment

HAOS running in HyperV virtual machine

### Search for similar issues

Yes

**Comment by xzessmedia** [comment]
i can report the same issue since a few days i have this problem also

**Comment by ddonevski** [comment]
There were previously reported issues with cloud password which appeared after firmware update. However, I'm also getting this error message and firmware was not updated. I set everything up a week ago and it worked fine for a couple of days while I was away. When I visited the house, I got this message with two C210 cams. A couple of days later, I visited the house again and then this also occured with C310 cam. I don't know how the software works so I suppose that error occurences just coincided with house visits. Anyway, all 3 of my cams can't be added to the integration because it refuses valid cloud password. I tried changing the password, didn't work. I didn't have time to factory reset the cameras, add them to Tapo app and then add then try adding them to the integration. Anyway, 3 cams are lost without touching anything.. Firmware, configurations, nothing was changed.
